### PR TITLE
Creating a Bot Account guide

### PR DESCRIPTION
**Pull request by One-Nub**

### CODE REVIEW

File: src/guides/creatingbotaccount.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (l7ssha) [inline src/guides/creatingbotaccount.md:1]: would be possible to add maybe some screenshots? if that wouldn't be that much of hassle

**Review by l7ssha — APPROVED** [review]

File: src/guides/creatingbotaccount.md (hunk 1 above)
Comment (One-Nub) [reply src/guides/creatingbotaccount.md:1]: sure - is there a preferred way to do that? i.e. image provider vs uploading to the repo

Comment (l7ssha) [reply src/guides/creatingbotaccount.md:1]: In repo would be more stable i guess. And should be achived pretty easily by adding to base

**Comment by l7ssha** [comment]
Gonna merge that PR since its done.
Reffering to https://github.com/nyxx-discord/nyxx_docs/pull/4#discussion_r716198120: Please open new PR to `main` :)
Thanks for contrib

**Pull request merged**